tox.ini

```diff
@@ -22,18 +22,22 @@ commands =
     pip install -e .[test]
     pytest --cov=hibpcli --cov=tests --cov-report term-missing  --cov-report html
 
-[testenv:flake8]
-deps =
-    flake8
-    flake8-click
-    flake8-bugbear
-commands =
-    flake8 src tests setup.py
+[testenv:flake8]	
+deps =	
+    flake8	
+    flake8-click	
+    flake8-bugbear	
+commands =	
+    flake8 src tests setup.py	
 
-[flake8]
-max-line-length = 88
+[flake8]	
+max-line-length = 88	
 ignore = E231  # clashes with black
```

Comment:
Please remove the trailing white spaces behind [flake8] and 88.

Reply:
removed whiteshpace